docs/model.er

```diff
@@ -75,6 +75,8 @@ info_url {label: "String"}
 +cloud_feeds {label: "Integer"}
 start_time {label: "Timestamp"}
 end_time {label: "Timestamp"}
+shoppinglist {label: "Integer"}
+upload_schedules {label: "Integer[]"}
```

Comment:
This is not a proper database relation. The upload_schedule should instead have a foreign key `campaign_id` that references the ID of a campaign.